RELEASE_NOTES.md

```diff
@@ -0,0 +1,102 @@
+# Performance Optimizations
+## Intel Architecture Processors
+* Improved matmul and inner product primitives performance on processors with Intel AMX instruction set support.
+* Improved performance of convolution and inner product primitives on processors with Intel AVX2 instruction set support.
+* Improved performance of `int8` convolution support with zero points.
+* Improved `fp32` convolution performance with `fp16` and `bf16` compressed weights on processors with Intel AVX2 or Intel AVX-512 instruction set support.
+* Improved `fp16`/`bf16` depthwise convolution performance with `fp32` bias or `sum` post-ops or dilation.
+* Improved `bf16` pooling backpropagation performance.
+* Improved binary post-ops performance with `per_w` broadcast.
+
+## Intel Graphics Products
+* Improved performance on Intel GPUs based on Xe3 architecture.
+* Improved convolution performance on:
+  * Intel Arc Graphics for Intel Core Ultra (Series 2, formerly Lunar Lake).
+  * Intel Arc B-series discrete graphics (formerly Battlemage).
+* Improved `int8` matmul performance with zero-points support for source and weight tensors.
+* Improved `f4_e2m1` and `f4_e3m0` matmul and reorder performance.
+* Improved performance of the following subgraphs with Graph API:
+  * [Scaled Dot Product Attention (SDPA)] with `int4` and `int8` [compressed key and value].
+  * `fp16`/`bf16` SDPA with `fp32` intermediate data types. Using `fp32` intermediate data types is recommended.
+  * SDPA with head size 512 and 576.
```

Comment:
Hi @syurkevi, could you please confirm if this is accurate?

Reply:
Yes, this was discussed in PR #2976.